testing/scenario_app/android/app/build.gradle

```diff
@@ -16,7 +17,7 @@ android {
         // NewerVersionAvailable and GradleDependency need to be taken care of
         // by a roller rather than as part of CI.
         // The others are irrelevant for a test application.
-        disable 'UnpackedNativeCode','MissingApplicationIcon','GoogleAppIndexingApiWarning','GoogleAppIndexingWarning','GradleDependency','NewerVersionAvailable','Registered'
+        disable 'UnpackedNativeCode','MissingApplicationIcon','GoogleAppIndexingApiWarning','GoogleAppIndexingWarning','GradleDependency','NewerVersionAvailable','Registered', 'ObsoleteLintCustomCheck'
```

Comment:
Can you add a comment why this was added?

Reply:
Hmm I can't remember at this point, as the original pr was a while ago. I'll try removing.